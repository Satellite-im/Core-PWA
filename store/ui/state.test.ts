import InitialUIState from '~/store/ui/state'

describe('init', () => {
  let inst: any

  beforeEach(() => {
    inst = InitialUIState()
  })

  it('should return the initial settings state', () => {
    expect(inst).toMatchSnapshot()
  })

  it('should not return the initial settings state', () => {
    expect(inst).not.toEqual({})
  })
})
