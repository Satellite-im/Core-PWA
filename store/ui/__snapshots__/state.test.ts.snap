// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`init should return the initial settings state 1`] = `
Object {
  "activeChannel": undefined,
  "chatImageOverlay": undefined,
  "chatbarContent": "",
  "chatbarFocus": false,
  "contextMenuPosition": Object {
    "x": 0,
    "y": 0,
  },
  "contextMenuStatus": false,
  "contextMenuValues": Array [],
  "editMessage": Object {
    "from": "",
    "id": "",
    "payload": "",
  },
  "enhancers": Object {
    "containerWidth": 0,
    "defaultHeight": "30rem",
    "defaultWidth": "24rem",
    "floating": false,
    "position": Array [
      0,
      0,
    ],
    "route": "emotes",
    "show": false,
  },
  "fileDownloadList": Array [],
  "filePreview": undefined,
  "fileSort": Object {
    "asc": true,
    "category": "modified",
  },
  "filesUploadStatus": "",
  "fullscreen": false,
  "glyphMarketplaceView": Object {
    "shopId": null,
    "view": "home",
  },
  "glyphModalPack": "",
  "hoveredGlyphInfo": undefined,
  "isReacted": false,
  "isScrollOver": false,
  "isTyping": false,
  "messages": Array [],
  "modals": Object {
    "callToAction": false,
    "changelog": false,
    "createServer": false,
    "creategroup": false,
    "crop": false,
    "error": false,
    "errorNetwork": Object {
      "action": null,
      "isOpen": false,
    },
    "glyph": false,
    "groupInvite": Object {
      "isOpen": false,
    },
    "marketplace": false,
    "newfolder": false,
    "quickchat": false,
    "renameFile": false,
    "userProfile": false,
    "wallet": false,
    "walletMini": false,
  },
  "mostEmojiUsed": Array [],
  "quickProfile": false,
  "quickProfilePosition": Object {
    "x": 0,
    "y": 0,
  },
  "recentGlyphs": Array [],
  "renameItem": undefined,
  "replyChatbarContent": Object {
    "from": "",
    "id": "",
    "payload": "",
  },
  "settingReaction": Object {
    "groupID": null,
    "messageID": null,
    "status": false,
  },
  "settingsRoute": "personalize",
  "settingsSideBar": true,
  "showMedia": false,
  "showOlderMessagesInfo": false,
  "showPinned": false,
  "showSearchResult": false,
  "showSettings": false,
  "showSidebar": true,
  "showSidebarUsers": true,
  "swiperSlideIndex": 0,
  "theme": Object {
    "base": Object {
      "class": "",
      "name": "default",
      "text": "Default",
      "value": "default",
    },
    "flair": Object {
      "text": "Satellite",
      "value": "#2761fd",
    },
  },
  "unreadMessage": 0,
  "userProfile": Object {},
}
`;
